src/renderers/webvr/WebVRManager.js

```diff
@@ -121,6 +126,18 @@ function WebVRManager( renderer ) {
 
 		poseObject.updateMatrixWorld();
 
+		var stageParameters = device.stageParameters;
+		
+		if ( stageParameters ) {
+
+			standingMatrix.fromArray( stageParameters.sittingToStandingTransform );
+			standingMatrixInverse.getInverse( standingMatrix );
+		
+			poseObject.matrixWorld.multiply( standingMatrix );
+			camera.matrixWorldInverse.multiply( standingMatrixInverse );
+		
+		}
+
```

Comment:
If no `stageParameters` we should create a `standingMatrix` just with `translate(0, scope.userHeight, 0)` and apply to the poseObject and camera too.

Reply:
Something like:
```
if (stageParameters) {
  standingMatrix.fromArray( stageParameters.sittingToStandingTransform );
  standingMatrixInverse.getInverse( standingMatrix );
        
  poseObject.matrixWorld.multiply( standingMatrix );
  camera.matrixWorldInverse.multiply( standingMatrixInverse );
} else {
  standingMatrix.identity().makeTranslation(0, scope.userHeight, 0);
  standingMatrixInverse.getInverse( standingMatrix );
        
  poseObject.matrixWorld.multiply( standingMatrix );
  camera.matrixWorldInverse.multiply( standingMatrixInverse );
}
```
I didn't test it though

Reply:
Well... you know
```
if (stageParameters) {
  standingMatrix.fromArray( stageParameters.sittingToStandingTransform );
} else {
  standingMatrix.identity().makeTranslation(0, scope.userHeight, 0);
}
  standingMatrixInverse.getInverse( standingMatrix );
  poseObject.matrixWorld.multiply( standingMatrix );
  camera.matrixWorldInverse.multiply( standingMatrixInverse );
```

Reply:
As the `standingMatrix` and the `standingMatrixInverse` won't be modified in the `!stageParams` case, you could just compute them once and just use them in that case, instead of getting the inverse and so on on each frame, but that's something that could be optimized later once we get it working ^_^

Reply:
scope.userHeight might change though

Reply:
@dmarcos aps, so in that case skip my last comment :-P